streamwebs_frontend/streamwebs/tests/models/test_water_quality.py

```diff
@@ -0,0 +1,124 @@
+from django.test import TestCase
+
+from django.contrib.gis.db import models
+from django.apps import apps
+from itertools import chain
+from django.core.exceptions import ValidationError
+
+from streamwebs.models import Site, Water_Quality, validate_wq_site
+
+
+class WaterQualityTestCase(TestCase):
+
+    def setUp(self):
+        self.expected_fields = {
+            'site_name': models.ForeignKey,
```

Comment:
shouldn't this just be 'site'? it will link to a whole site object, not just name